### PR TITLE
Github CI: Add step for running cxx-unittest

### PR DESCRIPTION
**Pull request by ibuclaw**
Mostly out of curiosity, but I did originally raise it as something that I'd like to be covered @Geod24.

### CODE REVIEW

File: .github/workflows/runnable_cxx.yml [hunk 1]
```diff
@@ -287,6 +287,16 @@ jobs:
           ./dmd/test/run.d runnable_cxx MODEL=32
         fi
 
+    ########################################
+    #      Run C++ frontend unittests      #
+    ########################################
+    - name: Run C++ frontend unittests
+      run: |
+        ./dmd/src/build.d cxx-unittest MODEL=64
```
Comment (MoonlightSentinel) [inline .github/workflows/runnable_cxx.yml:295]:
```suggestion
        ./dmd/src/build.d -j2 cxx-unittest MODEL=64
```

Ditto below

Comment (ibuclaw) [reply .github/workflows/runnable_cxx.yml:295]: I don't think any part of it can be parallelized...

Comment (MoonlightSentinel) [reply .github/workflows/runnable_cxx.yml:295]: Okay. Note that parallelization is enabled by default, this just limits the maximum job count.

Comment (ibuclaw) [reply .github/workflows/runnable_cxx.yml:295]: Ah, well the pipeline is sequential, so no risk of eating up our quota of cores here.

**Review by Geod24 — APPROVED** [review]

**Review by wilzbach — APPROVED** [review]

**Pull request merged**